docs/Compiler-Hardening-Guides/Compiler-Options-Hardening-Guide-for-C-and-C++.md

```diff
@@ -394,6 +396,52 @@ Some tools, such as `autoconf`, automatically determine what the compiler suppor
 
 ---
 
+### Warn on Unicode Unicode bidirectional (bidi) override characters |
+
+| [`-Wbidi-chars=any`](#-Wbidi-chars) | GCC | Warn on Unicode bidirectional (bidi) override characters |
```

Comment:
This table row seems to have been misplaced

Reply:
Fixed in d8347c193b383ed360d1d6b32ec049e150a18518

Reply:
Maybe "Source text solely left-to-right & GCC" ?